wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -208,8 +213,7 @@ addRecentPtxHistory wid currentHistory = do
 
 logTxHistory
     :: (Container t, Element t ~ TxHistoryEntry, WithLogger m, MonadIO m)
-    => Text -> t -> m ()
-logTxHistory desc =
-    logInfoS .
-    sformat (stext%" transactions history: "%listJson) desc .
-    map _thTxId . toList
+    => Text -> t -> m NoContent
+logTxHistory desc t = do
+    logInfoS . sformat (stext%" transactions history: "%listJson) desc . map _thTxId . toList $ t
+    return NoContent
```

Comment:
Seems strange that logging function returns some `NoContent`

Reply:
That's possibly an oversight during my refactoring, let me see if that was actually required and if not I agree, that's a mistake, it should really be `m ()` 😉